test/utils/data.js

```diff
@@ -752,7 +752,66 @@ export function project2000 () {
     }
   ]))
   .then(() => putObjectFromFile(bucket, 'scenario-2000/road-network_000000', FILE_ROAD_NETWORK))
-  .then(() => putObjectFromFile(bucket, 'scenario-2000/poi_000000', FILE_POI));
+  .then(() => putObjectFromFile(bucket, 'scenario-2000/poi_000000', FILE_POI))
+  .then(() => scenarioResults(
+    [
+      {
+        'id': 1,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200001,
+        'project_aa_id': 200001
+      },
+      {
+        'id': 2,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200002,
+        'project_aa_id': 200001
+      },
+      {
+        'id': 3,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200003,
+        'project_aa_id': 200001
+      }
+    ]
+  ))
+  .then(() => scenarioResultsPOI(
```

Comment:
- The files the analysis produces should be added as well.
- I've been using ids related to the scenario/project content belongs to, that's why they're all similar. Results ids should follow the same pattern.